src/main/java/task/Event.java

```diff
@@ -0,0 +1,40 @@
+package task;
+
+import utils.exceptions.IllegalValueException;
+import utils.formatters.Formatter;
+import utils.helpers.HelperFunctions;
+
+import java.time.LocalDateTime;
+
+public class Event extends Task {
+    private final LocalDateTime startDate;
+    private final LocalDateTime endDate;
+
+    public Event(String title, String startDate, String endDate) throws IllegalValueException {
+        this(title, startDate, endDate, false);
+    }
+
+    public Event(String title, String startDate, String endDate, boolean isDone) throws IllegalValueException {
+        super(title, isDone);
+        this.startDate = HelperFunctions.stringToDateTime(startDate);
+        this.endDate = HelperFunctions.stringToDateTime(endDate);
```

Comment:
Maybe you can consider handling time too since the package has those functionalities too